subsys/bluetooth/mesh/vnd/le_pair_resp.c

```diff
@@ -97,10 +97,11 @@ static void bt_mesh_le_pair_resp_reset(const struct bt_mesh_model *model)
 
 	bt_mesh_le_pair_resp_passkey_invalidate();
 
-	/* Clear all bonding information when receiving node reset */
-	err = bt_unpair(BT_ID_DEFAULT, NULL);
-	if (err) {
-		LOG_ERR("Unpair err: %d", err);
+	for (uint8_t id = 0; id < CONFIG_BT_ID_MAX; id++) {
```

Comment:
I think you need to use [bt_id_get](https://github.com/zephyrproject-rtos/zephyr/blob/476e5c5fc6946e5f0cea84717343fc3274419a59/include/zephyr/bluetooth/bluetooth.h#L406-L425) because not all identities may be configured.

Reply:
This may make it unnecesarily complicated. Usually it is 3 or 4 or 5 identities (and never 100 or 255). Unpair works in a way that if nothing exist, it is success. Even when there are 100+ identities compiled, it is still a rare event, so I wonder how much pendantic this removal code should be, and if it worth.

Reply:
Good point. This will avoid error logs for unconfigured identities.

Reply:
Sorry. I did not see the updated comments.